games-simulation/crrcsim/crrcsim-0.9.13.ebuild

```diff
@@ -35,11 +35,7 @@ src_prepare() {
 }
 
 src_configure() {
-	econf \
-		--datadir="/usr/share" \
-		--datarootdir="${EPREFIX}/usr/share" \
```

Comment:
`econf` doesn't pass `--datarootdir`. Have you checked that this isn't needed?

Reply:
verified, no change